controllers/users.js

```diff
@@ -346,15 +382,27 @@ class Users {
       });
     }
 
-    if (checkUser.id !== req.user.id) {
+    if (
+      checkUser.id !== req.user.id
+      && req.user.role !== 'super-admin'
+      && req.user.role !== 'admin'
+    ) {
```

Comment:
@shemaeric If one of these fails the whole condition will fail. If we are having admins and super-admins I suggest using **_or_**  Meaning the id is not equal or the role is not super admin or the role is not admin.